emulsion-palette: init at 3.3.9

foo-dogsquared opened the pull request:
###### Description of changes

Add [Emulsion](https://github.com/lainsce/emulsion), a tool to manage your color palettes. Fits nicely in a GNOME desktop.
<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

foo-dogsquared committed:
emulsion-palette: init at 3.3.9

azahi commented:
Result of `nixpkgs-review pr 182215` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>emulsion-palette</li>
  </ul>
</details>

azahi reviewed (approved):
Looks and works great!

IvarWithoutBones commented on pkgs/applications/graphics/emulsion-palette/default.nix:
```diff
@@ -0,0 +1,45 @@
+  ];
+
+  meta = with lib; {
+    mainProgram = "io.github.lainsce.Emulsion";
```
I think we should symlink the binary to just `emulsion`/`emulsion-palette` as well, seeing how this is a bit lengthy.

foo-dogsquared replied:
Done, seeing this is also how some applications like [Foliate](https://github.com/NixOS/nixpkgs/blob/c8ee69fb01f979e8980e1517c06eac8236776e85/pkgs/applications/office/foliate/default.nix) are packaged due to lengthy executable names. Though, I named the symlink `emulsion-palette` since there is [another package](https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/applications/graphics/emulsion/default.nix) with the same binary.

foo-dogsquared commented:
Rebased against `master` and dropped the addition to the maintainers list commit due to one of my [PRs](https://github.com/NixOS/nixpkgs/pull/182047) merged.

IvarWithoutBones reviewed (approved):
Looks good to me 👍

IvarWithoutBones commented:
Result of `nixpkgs-review pr 182215` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>emulsion-palette</li>
  </ul>
</details>
